### ISSUE TITLE
Wrong format of settings in ps cmdline

### ISSUE THREAD

**Issue by swingbit** [MonetDB/MonetDB#7548]
### Describe the bug

When looking at the `ps` output for `mserver5`, I see this strange format for settings: `--set=key value` (adding newlines here to improve readability):

```
mserver5 --dbpath=dbfarm/x
  --set=merovingian_uri mapi:monetdb://tachyon.hq.spinque.com:50100/x
  --set=mapi_listenaddr none
  --set=mapi_usock dbfarm/x/.mapi.sock
  --set=monet_vault_key dbfarm/x/.vaultkey
  --set=gdk_nr_threads 32
  --set=tablet_threads 32
  --set=max_clients 64
  --set=sql_optimizer sequential_pipe
  --set=embedded_py 3
```

Is this caused by the OS or by MonetDB?

I noticed the code in `forkmserver.c` consistently uses yet another strange format: `--set=key=value`  (2 `=`). I'm sure there is some good reason for this, but I wonder if it causes the issue I describe.

Why is this relevant? It's not a major problem, but it would be handy to have it displayed correctly  (this format isn't valid). For example, to copy&paste it as an argument to `valgrind`.

### Reproduction steps

```
ps ax | grep mserver5
```

### Expected behavior

Settings in the form: `--set key=value`

### MonetDB release

Dec2023

### Executables source

Self compiled and installed

### Operating System

Fedora 40

**Comment by swingbit** [comment]
It doesn't seem `ps`'s fault:
```
$ cat /proc/221401/cmdline | xargs -0 printf "%q|"; echo
mserver5|'--dbpath=dbfarm/x'|'--set=merovingian_uri'|mapi:monetdb://tachyon.hq.spinque.com:50100/x|'--set=mapi_listenaddr'|none|'--set=mapi_usock'|dbfarm/x/.mapi.sock|'--set=monet_vault_key'|dbfarm/x/.vaultkey|'--set=gdk_nr_threads'|32|'--set=tablet_threads'|32|'--set=max_clients'|64|'--set=sql_optimizer'|sequential_pipe|'--set=embedded_py'|3|
```

This shows that each key-value setting argument is seen as two separate arguments: `--set=key` and `value`

**Comment by sjoerdmullender** [comment]
mserver5 messes with its arguments (overwriting the equals sign separating the key and the value) in order to get two separate strings, one for the key, and one for the value.
--set takes one argument, which you can add as a separate command-line argument, or attached with an equals sign.  The argument should be a key=value combination, so you can indeed get two equals signs.
Looking at the ps output (or equivalently at /proc/$pid/cmdline) gives you the arguments after mserver5 messed with them.
By the way, this overwriting the equals sign is totally allowed: it's in the process's address space and is not read-only.
In other words, this is not a bug.
